versioned_docs/version-1.3-rc1(WIP)/depositpCKB.md

```diff
@@ -0,0 +1,76 @@
+---
+id: depositpCKB
+title: Add & Transfer pCKB on Godwoken v1
```

Comment:
> id: depositpCKB
title: Add & Transfer pCKB on Godwoken v1

Since the article is teaching users how to transfer pCKB, I suggest you to change `id` and `title` to fit the subject: Transfer pCKB. 
And then, you can organize and refactor part of the article's content around subject "transfer", which should give you a better sense of writing.

What do you think?